src/drawing/drawGoalCard.js

```diff
@@ -158,9 +158,11 @@ export default function render(
     // assume that it will be drawn the next time 'render' is called
     // if it isn't already set
     if (!img) return
+
     // adjust the x position according to the index of this member
     // since there can be many
-    const xImgDraw = x + goalWidth - (index + 1 * avatarWidth + avatarSpace)
+    const xImgDraw =
+      x + goalWidth - (index + 1) * (avatarWidth + avatarSpace / 2)
```

Comment:
can you go find `avatarSpace` in the code, and make it a few pixels less?

Reply:
and remove the `/ 2`

Reply:
ok